ch02.adoc

```diff
@@ -54,8 +54,8 @@ The examples in this document that use string-valued variables alternate between
 === Naming Conventions
 
 Variable, dimension, attribute and group names should begin with a letter and be composed of letters, digits, and underscores.
```

Comment:
Also clarify what is meant by  "a letter".

Reply:
@larsbarring, please submit that as a separate PR.  Thank you.